opentimelineio_contrib/adapters/advanced_authoring_format.py

```diff
@@ -263,6 +265,21 @@ def _transcribe(item, parent=None, editRate=24, masterMobs=None):
         # Does AAF support anything else?
         result.transition_type = otio.schema.TransitionTypes.SMPTE_Dissolve
 
+        # Extract value and time attributes of both ControlPoints used for
+        # creating AAF Transition objects
+        varying_value = None
+        for param in item.getvalue('OperationGroup').parameters:
+            if isinstance(param, aaf2.misc.VaryingValue):
+                varying_value = param
+                break
+
+        if varying_value:
```

Comment:
You might want to make this an `is None` check in case the truthiness of param ever gets built out in PyAAF2.

```suggestion
        if varying_value is None:
```